script.js

```diff
@@ -128,22 +127,22 @@ function pickUpItem(event) { //made in pickUpStuff
                             
                             var itemIsAboveBin = posY < curBin.y;
                             var itemIsInsideLeftWallOfBin = posX >= curBin.x;
-                            //console.log(curBin);
                             var itemIsInsideRightWallOfBin = ((posX+ items[i].element.offsetWidth) <= (curBin.x + curBin.element.offsetWidth));
                            
                             
                             if (!itemIsOnScreen || (!itemIsAboveBin && itemIsInsideLeftWallOfBin && itemIsInsideRightWallOfBin) ) {
                                if (!itemIsOnScreen){console.log('item is not on the screen');}
                                else{
                                 //snap in place
-                                posY += curBin.element.offsetHeight-items[i].element.offsetHeight - 5;
-                                console.log(posY);
+                                posY += curBin.element.offsetHeight-items[i].element.offsetHeight - curBin.element.offsetHeight/2;
                                 items[i].element.style.top = posY + 'px';
-                                console.log(items[i].element.style.top);
                                 console.log(`${i} is inside ${bin}`);
+                                items[i].inPackageId = `${bin}`;
+                                containers[bin].itemIndexInBin.push(i);
+                                console.log(containers[bin].itemIndexInBin);
```

Comment:
I haven't been super on top of commenting on these, but since we're entering the final bit of your project, probably worth cleaning up these unused `console.log`s and commented out code, just so things are a bit more tidy.